Symfony/CS/Fixer/Contrib/OrderedUseFixer.php

```diff
@@ -13,14 +13,67 @@
 namespace Symfony\CS\Fixer\Contrib;
 
 use Symfony\CS\AbstractFixer;
+use Symfony\CS\ConfigurationException\InvalidFixerConfigurationException;
 use Symfony\CS\Tokenizer\Tokens;
 
 /**
  * @author Sebastiaan Stok <s.stok@rollerscapes.net>
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author Darius Matulionis <darius@matulionis.lt>
  */
 class OrderedUseFixer extends AbstractFixer
 {
+    const SORT_ALPHA = 'alpha';
+
+    const SORT_LENGTH = 'length';
+
+    /**
+     * Sorting type.
+     *
+     * @var string
+     */
+    private static $sortType = self::SORT_ALPHA;
+
+    /**
+     * List of supported sorting types.
+     *
+     * @var array
+     */
+    private static $supportedSorters = array(self::SORT_ALPHA, self::SORT_LENGTH);
+
+    /**
+     * @param array $sortType
+     */
+    public static function configure(array $sortType = null)
+    {
+        // If no configuration was passed, stick to default.
```

Comment:
no configuration is null, everything else is a configuration, eg empty array is invalid config